runtime/pallene_core.h

```diff
@@ -266,4 +266,66 @@ TValue *pallene_getshortstr(Table *t, TString *key, size_t * restrict pos)
     }
 }
 
+/* To avoid looping infinitely due to integer overflow, Lua 5.4 carefully
```

Comment:
I think we should start to split this file into multiple files. We could have a single header called `pallene.h` that include all other headers.

Reply:
Right now we only have slightly over 10 functions and macros in pallene_core.h, so I don't feel an urgent need to split it up. I think the coder.lua is a bigger offender in this sense.

But I wouldn't be against it if someone comes up with a reasonable way to split the pallene_core.h into smaller files.

Reply:
There are a lot of lengthy files in Pallene source code; this header file isn't that long, but surely it could be broken down. To newcomers (and even myself) it is hard to navigate through these files. And most of the time these files could be easily divided because the functions are not interdependent.

By the way, you can merge this PR.

Reply:
Certainly. I don't know what would be the best way to split this particular file but I am open to suggestions.